community/examples/slurm-gke/slurm-gke.yaml

```diff
@@ -0,0 +1,192 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+blueprint_name: slurm-gke
+
+vars:
+  # The following variables should be over-written in the deployment.yaml file.
+  # Your GCP Project ID
+  project_id: ## Set GCP Project ID Here ##
+
+  # This should be unique across all of your Cluster
+  # Toolkit Deployments.
+  deployment_name: slurm-gke
```

Comment:
just fyi getting this error when deploying the blueprint as-is:
```

Error: Invalid value for variable

  on main.tf line 213, in module "slurm_controller":
 213:   slurm_cluster_name           = var.deployment_name
    ├────────────────
    │ var.slurm_cluster_name is "slurm-gke"

Variable 'slurm_cluster_name' must be a match of regex
'^[a-z](?:[a-z0-9]{0,9})$'.
```

Reply:
It maybe easier to pick a different default name, I think that validation has been around for a long time (so may be tricky to evaluate unintended side effects)